thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/detection/trigger/utils/DataAvailabilitySchedulingConfiguration.java

```diff
@@ -37,9 +37,11 @@
   private List<String> dataSourceWhitelist;
   private List<String> filterClassList;
   // delay time after each run for the scheduler to reduce DB polling
-  private long schedulerDelayInSec = 300; // 5 minute
+  private long schedulerDelayInSec = 300; // 5 minutes
   // default threshold if detection level threshold is not set
   private long taskTriggerFallBackTimeInSec = 24 * 60 * 60; // 1 day
+  // scheduling window for data availability scheduling
+  private long schedulingWindowInSec = 15 * 60; // 15 minutes
```

Comment:
Maybe it's better to use `TimeUnit.MINUTES.toSeconds(15)` than calculating seconds by multiplication.

Reply:
Good call!